examples/dreambooth/train_dreambooth_lora_hidream.py

```diff
@@ -1367,20 +1362,39 @@ def compute_text_embeddings(prompt, text_encoding_pipeline):
 
     vae_config_scaling_factor = vae.config.scaling_factor
     vae_config_shift_factor = vae.config.shift_factor
-    if args.cache_latents:
+
+    # if cache_latents is set to True, we encode images to latents and store them.
+    # Similarly to the pre-encoding in the case of a single instance prompt, if custom prompts are provided
+    # we encode them in advance as well (by iterating over training batches)
```

Comment:
Slightly less confusing I believe?

```suggestion
    # Similar to pre-encoding in the case of a single instance prompt, if custom prompts are provided
    # we encode them in advance as well.
```